features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/EnrichedResource.kt

```diff
@@ -32,7 +32,9 @@ internal data class EnrichedResource(
     }
 
     internal companion object {
-        internal const val APPLICATION_ID_KEY = "application_id"
+        internal const val APPLICATION_ID_RESOURCE_KEY = "applicationId"
```

Comment:
is there any impact from `application_id` to `applicationId` change?

Reply:
there's no impact, because this key is only for the EnrichedResource and not sent. Originally I used the same key as in the api, but the applicationId part of the api is no longer a single key